### ISSUE TITLE
[java] `DoNotUseThreads` reported for `Thread.currentThread().getName()`

### ISSUE THREAD

**Issue by Ledmington** [pmd/pmd#5468]
**Affects PMD Version:** 
Using PMD 7.9.0 from gradle.

**Rule:**
[DoNotUseThreads](https://pmd.github.io/pmd/pmd_rules_java_multithreading.html#donotusethreads).

**Description:**
I understand the motivation for this rule but I'm not creating/destroying/manipulating any threads. I am just accessing to the name of the current thread.

**Code Sample demonstrating the issue:**

```java
public class Main {
    public static void main(String[] args) {
        System.out.println(Thread.currentThread().getName());
    }
}
```

**Running PMD through:** *Gradle*

**Comment by adangel** [comment]
The rule was/is intended to be used in a JEE environment, such as a webservice etc. In that case, the webserver will manage the threads completely, so you won't need to do it yourself.

Question 1: Are you in this environment? If not, then maybe you don't need this rule at all?
Question 2: Why do you need to access the thread's name? Your app shouldn't need to be aware how it is executed, this is the task of the webserver...
